Gruntfile.js

```diff
@@ -17,7 +18,45 @@ module.exports = function(grunt) {
             "build/js/backend/NodeWebkit.js"
         ]
     };
-    
+
+    var buildConfig;
+    try {
+      buildConfig = require('./buildConfig.js');
+      console.log("buildConfig.js present - using it ...");
+    }
+    catch (noBuildConfigFound) {
+      console.log("no buildConfig.js present - using the standard configuration ...");
+      buildConfig = {
+        platforms:
+          [
+              "win32", "osx32", "linux32",
+              "win64", "osx64", "linux64"
+          ],
+        uglifyOptions:{}
+      };
+    }
+
+    console.log("Checking for dependencies ...");
+    try {
+      var bowerFD = fs.openSync("bower_components",'r');
+      fs.closeSync(bowerFD);
```

Comment:
Grunt also provides [grunt.file.exists()](http://gruntjs.com/api/grunt.file#grunt.file.exists) and [grunt.file.isDir()](http://gruntjs.com/api/grunt.file#grunt.file.isdir).